reference/v8.json

```diff
@@ -790,15 +791,19 @@
       "property-function": true,
       "values": [
         "map",
-        "viewport"
+        "viewport",
+        "auto"
       ],
-      "doc": "Aligns text to the plane of the `viewport` or the `map` when the map is pitched. Matches `text-rotation-alignment` if unspecified.",
+      "default": "auto",
+      "doc": "Aligns text to the plane of the `viewport` or the `map` when the map is pitched. Matches `text-rotation-alignment` if auto.",
```

Comment:
Nit: surround `auto` in backticks. (That way it’ll become `MGLTextPitchAlignmentAuto` when the iOS/macOS SDK documentation is automatically generated from this file.)